packages/plugin-css/src/index.ts

```diff
@@ -518,25 +529,38 @@ function getMode<T extends {id: string; $value: any; $extensions?: any; _origina
   return {value: token.$value, originalVal: token._original.$value};
 }
 
-/** reference an existing CSS var */
-export function varRef(id: string, tokens: ParsedToken[], generateName: ReturnType<typeof makeNameGenerator>, suffix?: string): string {
+/**
+ * Reference an existing CSS var
+ *
+ */
+export function varRef(
+  id: string,
+  options?: {
+    prefix?: string;
+    suffix?: string;
+    // note: the following properties are optional to preserve backwards-compat without a breaking change
```

Comment:
Does this mean they will be remove with the `const token = options?.tokens?.find((t) => t.id === refID);` find and warn bellow once there is a breaking change?

Reply:
We’re following [semver](https://semver.org/) in this package, so there shouldn’t be any breaking changes until `2.0.0`.

This was just an unintentional internal change that accidentally led to a breaking change. It was triggered by mismatching versions of plugin-css (+/- `1.6.0`) and plugin-sass (+/- `1.3.2`). This change just allows for (hopefully) graceful fallback between the two plugins. The optional chaining method just allows an older version of the plugin to omit that data, and it will, still render a result.

Reply:
Hidden cross-dependencies like this are an antipattern in general, and something I should clean up better. Just at the moment, it felt like saving work / reducing bugs to have plugin-sass rely on plugin-css, but over time they’ve become too tightly-coupled and I think there needs to be a common core they share (irrelevant for consumers; this is just an implementation detail I’m musing on).